src/golang/lib/databricks/jobs.go

```diff
@@ -0,0 +1,100 @@
+package databricks
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/clusters"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/databricks/databricks-sdk-go/service/libraries"
+	"github.com/dropbox/godropbox/errors"
+)
+
+func NewWorkspaceClient(
+	workspaceUrl string,
+	accessToken string,
+) (*databricks.WorkspaceClient, error) {
+	dConfig := &databricks.Config{
+		Host:  workspaceUrl,
+		Token: accessToken,
+	}
+	datatbricksClient, err := databricks.NewWorkspaceClient(dConfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "Unable to create Databricks client.")
+	}
+	return datatbricksClient, nil
+}
+
+func ListJobs(
+	ctx context.Context,
+	databricksClient *databricks.WorkspaceClient,
+) error {
```

Comment:
nit: any reason we don't return the job list here? It's a little strange for a method called `ListJobs` to not return any job info. The caller can just ignore it if they want. Or, if its only used for validating the integration, we can name it as such.